src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Telemetry/TelemetryScope.cs

```diff
@@ -19,9 +21,9 @@ internal sealed class TelemetryScope : IDisposable
     private readonly TimeSpan _minTimeToReport;
     private bool _disposed;
 
-    private TelemetryScope()
+    public TelemetryScope()
```

Comment:
It should be fine to remove this constructor now that `TelemetryScope` is a struct.

Reply:
I think we had a race condition in the PR.  Right around when you were reviewing, I was undoing the scope-as-struct change because of some of the warnings from having it NonCopyable with some of the use patterns.